src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -178,7 +213,16 @@ public void update(List<Endpoint> conferenceEndpoints)
             conferenceEndpoints = new ArrayList<>(conferenceEndpoints);
         }
 
-        long bweBps = Long.MAX_VALUE;
+        if (bweBps == -1)
+        {
+            bweBps = ((VideoMediaStream) dest.getStream())
+                .getOrCreateBandwidthEstimator().getLatestEstimate();
```

Comment:
Maybe we can use getRemoteBitrateEstimator here?
Also, I think some sanity check on the value from getLatestEstimate() might be useful, e.g. if (bweBps <0) bweBps = Long.MAX_VALUE